core/src/test/java/org/infinispan/statetransfer/StateTransferFunctionalTest.java

```diff
@@ -66,7 +65,7 @@ protected void createCacheManagers() throws Throwable {
             .useSynchronization(false)
             .recovery().disable();
       configurationBuilder.clustering().sync().replTimeout(30000);
-      configurationBuilder.clustering().stateTransfer().chunkSize(50);
+      configurationBuilder.clustering().stateTransfer().chunkSize(20);
```

Comment:
Nope. I just wanted to reduce the delay from 2 seconds to 1 second, and I tried to keep the number of `StateTransferResponseCommand`s roughly the same.